constantine/eth_verkle_ipa/common_utils.nim

```diff
@@ -6,22 +6,17 @@
 #   * Apache v2 license (license terms in the root directory or at http://www.apache.org/licenses/LICENSE-2.0).
 # at your option. This file may not be copied, modified, or distributed except according to those terms.
 
-## IPAConfiguration contains all of the necessary information to create Pedersen + IPA proofs
-## such as the SRS
 import
-  ./[eth_verkle_constants, barycentric_form],
+  ./eth_verkle_constants,
   ../platforms/primitives,
   ../math/config/[type_ff, curves],
-  ../math/elliptic/ec_twistededwards_projective,
+  ../math/elliptic/[ec_twistededwards_affine, ec_twistededwards_projective, ec_twistededwards_batch_ops],
   ../hashes,
   ../math/arithmetic,
-  ../math/elliptic/ec_scalar_mul,
-  ../math/elliptic/[ec_multi_scalar_mul, ec_multi_scalar_mul_scheduler],
-  ../platforms/[bithacks],
-  ../../research/kzg/strided_views,
```

Comment:
having to use `strided_views` from research folder was mentioned here:
- https://github.com/mratsim/constantine/pull/364#pullrequestreview-1926092368
- https://github.com/mratsim/constantine/pull/276#issuecomment-1869615664

Note that we still should use `Views` instead with `splitMiddle` as suggested in #367